src/main/kotlin/fi/metatavu/muisti/persistence/dao/ExhibitionPageDAO.kt

```diff
@@ -48,17 +47,26 @@ class ExhibitionPageDAO() : AbstractDAO<ExhibitionPage>() {
      * Lists ExhibitionPages by exhibition
      *
      * @param exhibition exhibition
+     * @param exhibitionDevice filter by exhibition device. Ignored if null is passed
      * @return List of ExhibitionPages
      */
-    fun listByExhibition(exhibition: Exhibition): List<ExhibitionPage> {
+    fun listByExhibition(exhibition: Exhibition, exhibitionDevice : ExhibitionDevice?): List<ExhibitionPage> {
```

Comment:
This method no longer lists just by exbition, so more appropriate name would be just lsit
```suggestion
    fun list(exhibition: Exhibition, exhibitionDevice : ExhibitionDevice?): List<ExhibitionPage> {
```